src/ride/ride_ratings.c

```diff
@@ -599,17 +599,214 @@ static int sub_65E72D(rct_ride *ride)
 	return edx & 0xFFFF;
 }
 
+static rating_tuple get_var_10E_rating(uint16 var_10E) {
+	int al;
+	al = var_10E >> 8;
+	al = al & 7;
+	int excitement = (al * 163840) >> 16;
+
+	al = var_10E >> 5;
+	al = al & 7;
+	excitement += (al * 196608) >> 16;
+
+	// Low bits, 0x0065DF3A
+	al = var_10E & 0x1F;
+	excitement += (al * 63421) >> 16;
+
+	al = var_10E >> 8;
+	al = al & 7;
+	int intensity = (al * 81920) >> 16;
+
+	al = var_10E >> 5;
+	al = al & 7;
+	intensity += (al * 49152) >> 16;
+
+	al = var_10E & 0x1F;
+	intensity += (al * 21140) >> 16;
+
+	al = var_10E >> 8;
+	al = al & 7;
+	int nausea = (al * 327680) >> 16;
+
+	al = var_10E >> 5;
+	al = al & 7;
+	nausea += (al * 204800) >> 16;
+
+	al = var_10E & 0x1F;
+	nausea += (al * 42281) >> 16;
+
+	rating_tuple rating = { excitement, intensity, nausea };
+	return rating;
+}
+
+/**
+ * rct2: 0x0065DF72
+ */
+static rating_tuple get_var_110_rating(uint16 var_110) {
+	int al;
+
```

Comment:
Repeat the above for this function as well. It will probably turn out later on that var_110 is a very similar thing to var_10E but at this point just create all new functions.